Documentation/user-guides/scrapeconfig.md

```diff
@@ -0,0 +1,162 @@
+---
+weight: 151
+toc: true
+title: ScrapeConfig CRD
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: true
+description: Guide to use ScrapeConfig to scrape targets external to the Kubernetes cluster
+---
+
+Starting with prometheus-operator v0.65.x, one can use the `ScrapeConfig` CRD to scrape targets external to the
+Kubernetes cluster or create scrape configurations that are not possible with the higher level
+`ServiceMonitor`/`Probe`/`PodMonitor` resources.
+
+# Prerequisites
+* `prometheus-operator` `>v0.65.1`
+* `ScrapeConfig` installed in the cluster. Make sure to restart the operator after having pushed the CRD.
+
+## Configure Prometheus or PrometheusAgent to select ScrapeConfigs
+
+Both the Prometheus and PrometheusAgent CRD have a `scrapeConfigSelector` field. This field needs to be set to a list of
+labels to match `ScrapeConfigs`:
+
+```yaml
+spec:
+  scrapeConfigSelector:
+    prometheus: system-monitoring-prometheus
+```
+
+With this example, all `ScrapeConfig` having the `prometheus` label set to `system-monitoring-prometheus` will be used
+to generate scrape configurations.
+
+# Use ScrapeConfig to scrape an external target
+
+`ScrapeConfig` currently supports a limited set of service discoveries:
+* `static_config`
+* `file_sd`
+* `http_sd`
+* `kubernetes_sd`
+
+Also, the following examples are basic and the CRD is constantly evolving, adding new features. Check the API documentation to see all the available fields.
+
+If you have an interest in another service discovery mechanism or you see something missing in the implementation, please
+[open an issue](https://github.com/prometheus-operator/prometheus-operator/issues).
+
+## `static_config`
+
+For example, to scrape a target located at `http://10.1.2.3:9100`, use the following:
+
+```yaml
+apiVersion: coreos.monitoring.com/v1alpha1
+kind: ScrapeConfig
+metadata:
+  name: my-scrape-config
+  namespace: my-namespace
+spec:
+  staticConfigs:
+    - labels:
+        job: mytarget
+      targets: 
+      - 10.1.2.3:9100
+    - labels:
+        job: prometheus
+      targets:
+        - prometheus.demo.do.prometheus.io:9090
```

Comment:
I like that we use the Prometheus demo instance as it's working out of the box. Do we need the other `10.1.2.3:9100` target?

Reply:
We don't, removed it 👍